fract4dgui/angle.py

```diff
@@ -33,18 +33,16 @@ def __init__(self, text, tip):
 
         self.adjustment.connect('value-changed', self.onAdjustmentValueChanged)
 
-        Gtk.DrawingArea.__init__(self, tooltip_text=tip)
-
-        self.set_size_request(40, 40)
+        Gtk.DrawingArea.__init__(self, tooltip_text=tip, width_request=40, height_request=40)
 
         self.set_events(
-            Gdk.EventMask.BUTTON_RELEASE_MASK |
-            Gdk.EventMask.BUTTON1_MOTION_MASK |
-            Gdk.EventMask.POINTER_MOTION_HINT_MASK |
-            Gdk.EventMask.ENTER_NOTIFY_MASK |
-            Gdk.EventMask.LEAVE_NOTIFY_MASK |
-            Gdk.EventMask.BUTTON_PRESS_MASK |
-            Gdk.EventMask.EXPOSURE_MASK
+            Gdk.EventMask.BUTTON_RELEASE_MASK
```

Comment:
I'm intrigued as to your choice here.. the code base consistently puts the |'s on the right at the moment, and I for one find this much easier to read - while the at the beginning not so much.

The consistency point is of more interest to me here though.

Reply:
PEP 8 changed (for new code) five years ago on this:
https://github.com/python/peps/commit/7690a00d478866eb3cae3d82bbe1e80199a7392f#diff-db417657713fab81e79cb425b07ca55ecb11fc01fc52627e2756d450e8ac6f04

apparently to follow mathematical convention as recorded by Donald Knuth.

I only changed it because `flake8` was complaining - but I must of had a global setting in place because it is OK now. I'll change them back, I don't want to make it harder to read.

Reply:
Those are all done now in a new commit.